pkg/istio/rbac/v2/provider_test.go

```diff
@@ -18,15 +22,13 @@ import (
 	k8sv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/cache"
-	"sort"
-	"testing"
-	"time"
 )
 
 const (
 	domainName       = "test.namespace"
 	username         = "user.name"
 	wildcardUsername = "user.*"
+	wildcardAllUsers = "test-domain.namespace2.*"
```

Comment:
Could we also rename this to `wildcardAllDomainUsers` or something along those lines?

Reply:
https://github.com/yahoo/k8s-athenz-istio-auth/pull/58/commits/db86ddc3af039aebaa3d8d72ec673964855b4ce2